pkg/eventcollector/eventcollector.go

```diff
@@ -58,7 +58,7 @@ type EventCollector struct {
 
 	triggers []Trigger
 
-	filterTriggers []model.ContractFilterers
+	additionalNewsroomFilterers []model.ContractFilterers
```

Comment:
@petersng , are you ok with this new naming? Or should I do the `filterTriggers` and `watcherTriggers`